pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -597,13 +599,41 @@ func (m *kubeGenericRuntimeManager) killContainer(pod *v1.Pod, containerID kubec
 	return err
 }
 
+func isSidecar(containers []v1.Container, name string) bool {
```

Comment:
can we move this method to podutil.go so it can be commonly shared by various components?

Reply:
I don't know if it's wise to put a helper for an alpha feature there, it will be easier to remove this feature if we keep it all close together. Other components shouldn't be using this feature yet. It wouldn't be difficult to move it there once this got promoted to beta/ga

Reply:
That’s a good point! Not sure what’s the convention here, might wanna check with package owners.